src/components-examples/material/slider/slider-configurable/slider-configurable-example.css

```diff
@@ -12,44 +13,32 @@
 .example-margin {
   margin: 8px;
 }
+
 .example-width {
   max-width: 180px;
   width: 100%;
 }
 
-.mat-slider-horizontal {
+.mat-mdc-slider {
   max-width: 300px;
   width: 100%;
 }
 
-.mat-slider-vertical {
-  height: 300px;
-}
-
-.mat-card + .mat-card {
+.mat-mdc-card + .mat-mdc-card {
   margin-top: 8px;
 }
 
 .example-result-card h2 {
   margin: 0 8px;
 }
 
-.example-result-card .example-label-container.example-label-horizontal {
+.example-label-container {
   display: flex;
   justify-content: space-between;
-  margin: 0 0 12px 16px;
+  margin: 20px 10px 0;
```

Comment:
Google-convention is usually to stick with an 8px system, so I'd suggest this to be something like "16px 8px 0"

Reply:
I ended up with this number so that it aligns with some of the other elements on the page.